DemoApp/src/main.cc

```diff
@@ -1,34 +1,117 @@
 #include <iostream>
+#include <sstream>
 
 #include "Echo.h"
+#include "Utils.h"
 #include "views.h"
 
+namespace {
+std::vector<std::string> getOptions(int argc, char *argv[]) {
+    std::vector<std::string> res;
+    for (int i = 1; i < argc; i++) {
+        res.emplace_back(argv[i]);
+    }
+
+    return res;
+}
+
+bool startsWith(const std::string &s, const std::string &prefix) {
+    if (s.length() < prefix.length()) {
+        return false;
+    }
+
+    for (size_t i = 0; i < prefix.length(); i++) {
+        if (s[i] != prefix[i]) {
+            return false;
+        }
+    }
+
+    return true;
+}
+
+std::vector<std::string> split(const std::string &s, char delimiter) {
+    std::vector<std::string> tokens;
+    std::string token;
+    std::istringstream tokenStream(s);
+    while (std::getline(tokenStream, token, delimiter)) {
+        tokens.push_back(token);
+    }
+    return tokens;
+}
+}  // namespace
+
+std::map<std::string, std::string> optionsMap(const std::vector<std::string> &args) {
```

Comment:
A to też nie powinno być w anonimowym namespace?

Reply:
Poprawione